relayer/src/link/relay_path.rs

```diff
@@ -565,7 +575,7 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
         &self,
         initial_odata: OperationalData,
     ) -> Option<OperationalData> {
-        info!(
+        warn!(
```

Comment:
Could this benefit from the tracking ID of `initial_odata` under the same `id` key as elsewhere?

Reply:
> Could this benefit from the tracking ID of initial_odata under ..

Hmm I'm not sure what you mean by "this". Do you mean the `self` object, which is of type `RelayPath`? I think for this type of object we don't have tracking `id`s.

On line 586 below, the tracking `id` is propagated from `initial_data` into new `OperationalData` objects, which happens in method `generate_operational_data`.